docs/tutorials/getting-started.md

```diff
@@ -236,7 +236,7 @@ The following optional values are supported by `<py-config>`:
   * runtimes (List of Runtimes): List of runtime configurations. Each Runtime expects the following fields:
     * src (string, Required): URL to the runtime source.
     * name (string): Name of the runtime. This field can be any string and is to be used by the application author for their own customization purposes.
-    * name (string): Programming language supported by the runtime. This field can be used by the application author to provide clarify. It currently has no implications on how PyScript behaves.
+    * lang (string): Programming language supported by the runtime. This field can be used by the application author to provide clarify. It currently has no implications on how PyScript behaves.
```

Comment:
Good catch! :)
"to provide clarify" should be "to provide clarification" I guess?
If you want, you could add another commit for that. :)

Reply:
Oh, I skipped that. Thanks a lot for pointing it out. I made another commit for that.

Reply:
Great, thank you!